src/MicrosoftTeams.ts

```diff
@@ -79,7 +79,9 @@ namespace microsoftTeams
     {
         if (initializeCalled)
         {
-            throw new Error("initialize must not be called more than once.");
```

Comment:
Do we still need to throw here if we expect developers to do this? I think we should just get rid of this exception.